### ISSUE TITLE
How to inference on Lister with my own images?

### ISSUE THREAD

**Issue by lzzzzj** [AlibabaResearch/AdvancedLiterateMachinery#84]
It seems like I can test only with lmdb dataset.
(python test.py -c=config/lister.yml --model_name=lister_base)
Is there some way I can inference with only images?

**Comment by ccx1997** [comment]
Of course. Refer to the data preprocessing in `dataset/dataset.py` in LISTER, to load image data in your custom way, or convert your images into lmdb format following [here](https://github.com/FangShancheng/ABINet/blob/main/tools/create_lmdb_dataset.py).

By the way, you can assign the label as a dummy string like 'aaaaaaaaaa'. If label is not given, the argument `max_char` in [this line](https://github.com/AlibabaResearch/AdvancedLiterateMachinery/blob/main/OCR/LISTER/test.py#L43C17-L43C17) should be deleted. The argument `max_char` placed here is just to avoid unnecessary character map generation during evaluation.

**Comment by lzzzzj** [comment]
> Of course. Refer to the data preprocessing in `dataset/dataset.py` in LISTER, to load image data in your custom way, or convert your images into lmdb format following [here](https://github.com/FangShancheng/ABINet/blob/main/tools/create_lmdb_dataset.py).
> 
> By the way, you can assign the label as a dummy string like 'aaaaaaaaaa'. If label is not given, the argument `max_char` in [this line](https://github.com/AlibabaResearch/AdvancedLiterateMachinery/blob/main/OCR/LISTER/test.py#L43C17-L43C17) should be deleted. The argument `max_char` placed here is just to avoid unnecessary character map generation during evaluation.

Problem solved.
Thanks a lot!
